src/addons/build.js

```diff
@@ -174,56 +174,56 @@ export default (XRegExp) => {
             // Named subpattern
             if (subName) {
                 if (!data.hasOwnProperty(subName)) {
-                    throw new ReferenceError('Undefined property ' + $0);
+                    throw new ReferenceError(`Undefined property ${$0}`);
                 }
                 // Named subpattern was wrapped in a capturing group
                 if ($1) {
                     capName = outerCapNames[numOuterCaps];
                     outerCapsMap[++numOuterCaps] = ++numCaps;
                     // If it's a named group, preserve the name. Otherwise, use the subpattern name
                     // as the capture name
-                    intro = '(?<' + (capName || subName) + '>';
+                    intro = `(?<${capName || subName}>`;
                 } else {
                     intro = '(?:';
                 }
                 numPriorCaps = numCaps;
-                return intro + data[subName].pattern.replace(subParts, (match, paren, backref) => {
+                return `${intro + data[subName].pattern.replace(subParts, (match, paren, backref) => {
```

Comment:
Yeah, I was a bit iffy on that one too... I just pushed https://github.com/slevithan/xregexp/pull/194/commits/b1def280d73a8a6ec96b6b69d2499426c0525037 implementing the suggestion of an extra variable. Let me know what you think.